Makefile

```diff
@@ -154,3 +159,6 @@ ifneq ($(CLOUD_PROVIDER),)
 else
 	$(error CLOUD_PROVIDER is not set)
 endif
+ifeq ($(RESOURCE_CTRL),true)
+	$(MAKE) -C ./peerpod-ctrl undeploy
```

Comment:
Don't you need to delete on the inverse order of apply? i.e. delete the ctrl first then peer pods?